### DISCUSSION TITLE
Add "Allowed" filter to Firewall -> All Apps screen

### DISCUSSION THREAD

**Discussion by indeed-a-genius** [celzero/rethink-app#529] [Ideas]
I Exclude most of my apps from Firewall (another idea: option for new apps to be excluded by default)
I see no way to show on the All Apps screen only the apps that are Allowed (so not Excluded).

**Comment by ignoramous** [comment]
>  I see no way to show on the All Apps screen only the apps that are Allowed (so not Excluded).

We've added this feature based on your feedback in `v053k` (:
